components/server/src/api/configuration-service-api.ts

```diff
@@ -56,8 +56,8 @@ export class ConfigurationServiceAPI implements ServiceImpl<typeof Configuration
         private readonly apiConverter: PublicAPIConverter,
         @inject(UserService)
         private readonly userService: UserService,
-        @inject(ContextService)
```

Comment:
@filiptronicek Can we keep using `ContextService`, but just use/make up another method? `parseContextUrl` looks fitting, no?

Reasoning:
 1. Let's not change more then we need
 2. It's not clear whether there is value in this change, as lowers the abstraction of an import.
 3. We actually use the `normalizeContextURL` call in between which is quite important

Reply:
> parseContextUrl seems fitting indeed, but is marked as deprecated

Just removed that mark if it does :100: of what we want.

Reply:
Changed in [`8d6526e` (#20264)](https://github.com/gitpod-io/gitpod/pull/20264/commits/8d6526e113a721459ab7a1ab87a0dcf9dd222845)